pandas/core/algorithms.py

```diff
@@ -15,6 +15,7 @@
 from warnings import warn
 
 import numpy as np
+import numpy.typing as npt
```

Comment:
import npt from pandas._typing (it is guarded). but ok to use `import numpy.typing as npt` in stubs.